mlir/utils/jenkins/RockDriver.py

```diff
@@ -29,13 +29,13 @@
 DATA_TYPES = ['conv', 'convfp16', 'convint8']
```

Comment:
In this file, `MIOpen` refers to the MIOpen project instead of the dialect, except for mlir tools `mlir-miopen-driver` and `miopen-gen`.